### PR TITLE
docs(beginners,advanced): improve grammar in several doc pages

### PR DESCRIPTION
**Pull request by pwagener**
This improves the sentence structure in various places of the documentation.
It does not change the content, and avoids changing the already-excellent
sample code in the docs.

### CODE REVIEW

File: docs/bootstrap.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 
-After creating our resolvers and types classes with other business-related code, we need to make our app running. At first we have to build the schema, then we can expose it by HTTP server, WebSockets or even MQTT.
+After creating our resolvers, types classes, and other business-related code, we need to make our app run. First we have to build the schema, then we can expose it by HTTP server, WebSockets or even MQTT.
 
-## Create executable schema
+## Create Executable Schema
 
-To create executable schema from types' and resolvers' definitions, you need to use `buildSchema` function.
-It takes configuration object as a parameter and returns a promise of `GraphQLSchema` object.
+To create an executable schema from types and resolver definitions, you need to use the `buildSchema` function.
```
Comment (19majkel94) [inline docs/bootstrap.md]: There are multiple type classes and resolver classes, so I think that it should be "from type and resolver definitions"? 😕

File: docs/bootstrap.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ bootstrap(); // actually run the async function
 
 ## Create HTTP GraphQL endpoint
 
-In most cases, the GraphQL app is served by a HTTP server. After building the schema we can create it using e.g. [`apollo-server`](https://github.com/apollographql/apollo-server) package:
+In most cases, the GraphQL app is served by a HTTP server. After building the schema we can create it using the [`apollo-server`](https://github.com/apollographql/apollo-server) package:
```
Comment (19majkel94) [inline docs/bootstrap.md]: `apollo-server` is just one of the tools that you can use - earlier the examples used `graphql-yoga`. I would like to underline that fact.

**Comment by pwagener** [comment]
@19majkel94 Good suggestions -- thank you for the review!  I tweaked those two sentences to incorporate the feedback.

Overall the documentation is excellent; I'm happy I can improve it a bit and am looking forward to finding time to contribute to this project.

**Comment by 19majkel94** [comment]
@pwagener 
For future contributions - please don't force push amended commits as it makes checking changes after review so much harder. I always squash PR with proper merge commit message so don't afraid of bad commit history with names like `PR Fix` for the commits 😉

**Pull request merged**